devicetypes/axis/axis-gear-v5.1.src/axis-gear-v5.1.groovy

```diff
@@ -0,0 +1,420 @@
+        command "ShadesUp"
+        command "ShadesDown"
+        
+        // command to stop blinds
+        command "stop"
+        command "getversion"
+        
+        fingerprint profileID: "0104", manufacturer: "AXIS", model: "Gear", deviceJoinName: "AXIS Gear"
+        fingerprint profileId: "0104", deviceId: "0202", inClusters: "0000, 0003, 0006, 0008, 0102, 0020, 0001", outClusters: "0019", manufacturer: "AXIS", model: "Gear", deviceJoinName: "AXIS Gear"
+        fingerprint endpointID: "01, C4", profileId: "0104, C25D", deviceId: "0202", inClusters: "0000, 0003, 0006, 0008, 0102, 0020, 0001", outClusters: "0019", manufacturer: "AXIS", model: "Gear", deviceJoinName: "AXIS Gear"
+        //fingerprint endpointID: "C4", profileId: "C25D", inClusters: "0001", outClusters: "0001", deviceJoinName: "AXIS Gear2"
+
+        //ClusterIDs: 0000 - Basic; 0006 - On/Off; 0008 - Level Control; 0102 - Window Covering;
+        //Updated 2017-06-21
+        //Updated 2017-08-24 - added power cluster 0001 - added battery, level, reporting, & health check
+        //Updated 2018-01-04 - Axis Inversion & Increased Battery Reporting interval to 1 hour (previously 5 mins)
+        //Updated 2018-01-08 - Updated battery conversion from [0-100 : 00 - 64] to [0-100 : 00-C8] to reflect firmware update
+        //Updated 2018-11-01 - added in configure reporting for refresh button, close when press on partial shade icon, update handler to parse between 0-254 as a percentage
+        //Updated 2019-06-03 - modified to use Window Covering Cluster Commands and versioning tile and backwards compatibility (firmware and app), fingerprinting enabled
+    }
+   
+	tiles(scale: 2) {
+        multiAttributeTile(name:"windowShade", type: "lighting", width: 3, height: 3) {
+            tileAttribute("device.windowShade", key: "PRIMARY_CONTROL") {
+                attributeState("open",  label: 'Open',  action:"close", icon:"http://i.imgur.com/4TbsR54.png", backgroundColor:"#ffcc33", nextState: "closing")
+                attributeState("partially open", label: 'Partially Open', action:"close", icon:"http://i.imgur.com/vBA17WL.png", backgroundColor:"#ffcc33", nextState: "closing")
+                attributeState("closed", label: 'Closed', action:"open",  icon:"http://i.imgur.com/mtHdMse.png", backgroundColor:"#bbbbdd", nextState: "opening")
+                attributeState("opening", label: 'Opening', action: "stop", icon: "http://i.imgur.com/vBA17WL.png", backgroundColor: "#ffcc33", nextState: "stopping")
+                attributeState("closing", label: 'Closing', action: "stop", icon: "http://i.imgur.com/vBA17WL.png", backgroundColor: "#bbbbdd", nextState: "stopping") 
+                attributeState("stopping", label: 'Stopping',  icon: "http://i.imgur.com/vBA17WL.png", backgroundColor: "#ff7777") 
+                attributeState("stoppingNS", label: 'Stopping Not Supported',  icon: "http://i.imgur.com/vBA17WL.png", backgroundColor: "#ff7777") 
```

Comment:
These states "stopping" and "stoppingNS" will not be visible in the new app and behavior may be strange because of it.